src/main/java/com/aws/greengrass/clientdevices/auth/connectivity/ConnectivityInformation.java

```diff
@@ -66,22 +64,20 @@ public List<String> getCachedHostAddresses() {
      * Get connectivity info.
      *
      * @return list of connectivity info items
+     * @throws software.amazon.awssdk.services.greengrassv2data.model.GreengrassV2DataException
+     * if getConnectivityInfo call fails
      */
     public List<ConnectivityInfo> getConnectivityInfo() {
         GetConnectivityInfoRequest getConnectivityInfoRequest =
                 GetConnectivityInfoRequest.builder().thingName(Coerce.toString(deviceConfiguration.getThingName()))
                         .build();
         List<ConnectivityInfo> connectivityInfoList = Collections.emptyList();
 
-        try {
-            GetConnectivityInfoResponse getConnectivityInfoResponse =
-                    clientFactory.getGreengrassV2DataClient().getConnectivityInfo(getConnectivityInfoRequest);
-            if (getConnectivityInfoResponse.hasConnectivityInfo()) {
-                // Filter out port and metadata since it is not needed
-                connectivityInfoList = getConnectivityInfoResponse.connectivityInfo();
-            }
-        } catch (ValidationException | ResourceNotFoundException e) {
```

Comment:
and why are we now removing error handling from here?

Reply:
reason is on other comment, moved the handling to CISShadowMonitor

Reply:
maybe it'd be better to make the return type an optional, so the error handling stays here

Reply:
adjusted in latest commit